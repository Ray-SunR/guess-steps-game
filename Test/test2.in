gam 	
gama
		


game 1 1 test2A1.txt test2B1.txt
		

game 2 2 test2A2.txt test2B2.txt
quia
quit
