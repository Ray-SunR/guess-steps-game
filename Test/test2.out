Total is 1, max deduction is 1
Player A's move
Total is 0
Player A wins
Score is
A 1
B 0
Total is 2, max deduction is 2
Player B's move
Total is 0
Player B wins
Score is
A 1
B 1
